### ISSUE TITLE
Great job! I have a few questions.

### ISSUE THREAD

**Issue by zhoushen1** [qyp2000/XPSR#3]
Hello! I’m trying to reproduce your code, but I'm currently facing some issues and would appreciate your help!

1. Could you guide me on what the dataset directory structure should look like (how should the training and testing data be organized)?
2. I’d like to know what the directory structure for the pre-prepared `highlevel_prompt` and `lowlevel_prompt` results should look like before starting the actual training. Should it be structured like the example below?
![微信截图_20240828122421](https://github.com/user-attachments/assets/04a55696-909c-42b9-8da4-e3b02e196c6d)

**Comment by qyp2000** [comment]
The dataset structure of the training set is roughly shown in the following figure 1.
The number of the dataset represents the number of times we sampled (randomly cropped) from the original image. Below is the HR/LR/prompt folder, which includes files from different datasets, such as DIV2k/FFHQ/OST, according to the dataset.
The test set does not require randomly cropped sampling, and its structure is shown in the figure 2, which includes different samples in the HR/LR/prompt folder in the DrealSR dataset
![image (1)](https://github.com/user-attachments/assets/c48cb44d-490d-40bd-8bf4-43639b7d3a70)
![image](https://github.com/user-attachments/assets/2587b701-78cc-4d8d-99c4-d24744ab6cd0)
